Jetcaster/app/src/main/java/com/example/jetcaster/data/PodcastsRepository.kt

```diff
@@ -60,6 +61,9 @@ class PodcastsRepository(
                         }
                     }
                 }
+                } catch (e: Throwable) {
+                    println("podcastsFetcher(SampleFeeds).collect error: $e")
```

Comment:
Can we do better than this? If not, let's use at least `Log.d` to print the message

Reply:
It needs more UI work that will be pushed after. For now we have a dialog when the app is opened